lib/squash.js

```diff
@@ -29,7 +30,11 @@ const squash = (args) => {
     Util.createDirectory();
   }
 
-  FileOps.createCommand(alias, command);
+  if (Flags.parseFlags(flags).indexOf(CONSTANTS.FLAGS.PASS_ARGS) > -1) {
```

Comment:
Let's move this in `util.js` as a function, with name something like `formatCommand` to something else
This function should handle all the command modification based on INPUT flags.
Also let's make it a switch case, so it will be easier if we want to add more flags later.

Reply:
I moved it to Util.formatCommand(). And added a test for it.